src/geometry/manifold/charted_space.lean

```diff
@@ -489,7 +489,7 @@ atlas members are just the identity -/
 by simp [atlas, charted_space.atlas]
 
 /-- In the model space, chart_at is always the identity -/
-@[simp, mfld_simps] lemma chart_at_self_eq {H : Type*} [topological_space H] {x : H} :
+lemma chart_at_self_eq {H : Type*} [topological_space H] {x : H} :
```

Comment:
Is there a reason to drop the `@[simp, mfld_simps]` here?

Reply:
Because the lints wants me to after I added the lemma `local_homeomorph.singleton_charted_space_chart_at_eq`, as it magically recognizes that `charted_space_self` is an instance of `singleton_charted_space`. I had to take off many simps,  was undecided if to take them off or if to higher the priority, since one now needs 4 or 5 lemmas instead of one to perform the same simplification. What is the standard practice in such cases?

Reply:
Interesting!  I don't know, hopefully someone else does.

Reply:
@sgouezel ?